var/spack/repos/builtin/packages/gaudi/package.py

```diff
@@ -146,6 +150,10 @@ def cmake_args(self):
             # todo:
             self.define("GAUDI_USE_INTELAMPLIFIER", False),
         ]
+        # Release notes for v39.0: https://gitlab.cern.ch/gaudi/Gaudi/-/releases/v39r0
+        # Gaudi@39: needs C++ >= 20, and we need to force CMake to use C++ 20 with old gcc:
+        if self.spec.version >= Version("39.0") and self.compiler.version < Version("13.0.0"):
```

Comment:
```suggestion
        if self.spec.satisfies("@39: %gcc@:13"):
```
I think this should work (and at least at some point was recommended to me over using Version comparisons directly). Technically, we might even lower this to `@gcc:12`, because I could build it without specifying a c++ version with `gcc@12.3`. However, it also doesn't really hurt explicitly specifying it in that case.